CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
 
 namespace CompanyEmployees.Core.Domain.Entities
```

Comment:
Consider marking `Salary` as nullable (`decimal?`) if it is optional, or add validation attributes like `[Required]` to enforce presence. Also, consider adding XML documentation to clarify its purpose.

```suggestion
public decimal? Salary { get; set; }
```